web/src/components/molecules/Asset/AssetList/AssetListHeader/index.tsx

```diff
@@ -1,27 +1,45 @@
 import styled from "@emotion/styled";
 
-import { UploadChangeParam, UploadFile } from "@reearth-cms/components/atoms/Upload";
+import { UploadProps, UploadFile } from "@reearth-cms/components/atoms/Upload";
 import UploadAsset from "@reearth-cms/components/molecules/Asset/UploadAsset";
 
 type Props = {
   title: string;
   subTitle: string;
-  handleUpload: (info: UploadChangeParam<UploadFile<any>>) => void;
+  fileList: UploadFile<any>[];
+  uploading: boolean;
+  uploadProps: UploadProps;
+  uploadModalVisibility: boolean;
+  displayUploadModal: () => void;
+  hideUploadModal: () => void;
+  handleUpload: () => void;
 };
 
-const AssetListHeader: React.FC<Props> = ({ title, subTitle, handleUpload }) => {
+const AssetListHeader: React.FC<Props> = ({
+  title,
+  subTitle,
+  fileList,
+  uploading,
+  uploadProps,
+  handleUpload,
+  uploadModalVisibility,
+  displayUploadModal,
+  hideUploadModal,
```

Comment:
```suggestion
  uploadModalVisibility,
  displayUploadModal,
  hideUploadModal,
  handleUpload,
```

Please try to retain the same order as the props type so its easier to read, update and spot any unused props in the type (often happens..)